Cython/Compiler/ExprNodes.py

```diff
@@ -9368,7 +9368,8 @@ def generate_cyfunction_code(self, code):
         elif def_node.is_classmethod:
             flags.append('__Pyx_CYFUNCTION_CLASSMETHOD')
 
-        if def_node.local_scope.parent_scope.is_c_class_scope and not def_node.entry.is_anonymous:
+        if (def_node.local_scope.parent_scope.is_c_class_scope
+            and not def_node.entry.is_anonymous):
```

Comment:
I don't see a reason for this change. (We allow 120 characters per line in Cython and usually accept up to 150.)
```suggestion
        if def_node.local_scope.parent_scope.is_c_class_scope and not def_node.entry.is_anonymous:
```